### PR TITLE
2940 twig title and controls

### PR DESCRIPTION
**Pull request by sinejespersen**
#### Link to ticket

https://leantime.itkdev.dk/dashboard/show#/tickets/showTicket/2940

#### Description

- Create a [base file](https://github.com/itk-dev/ai-screening/compare/feature/2940-twig-title-and-controls?expand=1#diff-f1770f9db0ae968f6e74a0d5fb3534bc86a0ef635f29244cb527224e4b68e6fe) for the title (with title and "controls")
- Changed `views-view.html.twig` to `views-view--projects.html.twig`

#### Screenshot of the result

n/a

### CODE REVIEW

File: web/themes/custom/itkdev/itkdev_project_theme/templates/content-wrapper.html.twig [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    </div>
+  {% endblock %}
+
+</div>
+{% block content %}
+  <div{{ attributes }}>
+    {{ content }}
+  </div>
+{% endblock %}
+{{ dump() }}
```
Comment (sinejespersen) [inline web/themes/custom/itkdev/itkdev_project_theme/templates/content-wrapper.html.twig:30]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,10 +17,13 @@ and this project adheres to [Semantic Versioning].
   OIDC tweaks
 - [PR-32](https://github.com/itk-dev/ai-screening/pull/32)
   2938: Use x icon for navbutton when navigation is opened
-- [PR-29](https://github.com/itk-dev/ai-screening/pull/29)
-  2937: Entity access check updates
+- [PR-31](https://github.com/itk-dev/ai-screening/pull/31)
+  Create a base filefor the title (with title and "controls")
```
Comment (sinejespersen) [inline CHANGELOG.md:21]:
```suggestion
  Create a base file for the title (with title and "controls")
```

Comment (sinejespersen) [inline CHANGELOG.md:21]: More here?

**Review by sinejespersen — COMMENTED** [review]
Perhaps more additions to changelog?

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: web/modules/custom/ai_screening/src/Helper/ThemeHelper.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  }
+
+  /**
+   * Enable node edit form template.
+   */
+  public function formAlter(FormAlterEvent $event): void {
+    $form = &$event->getForm();
+    $formId = $event->getFormId();
+    if ('node_project_edit_form' == $formId) {
+      $form['#theme'] = 'node_edit_form';
```
[the remark this answers is not in the document]
Comment (martinyde) [reply web/modules/custom/ai_screening/src/Helper/ThemeHelper.php:35]: I'm not quite sure myself, but it is required for the template to work:
https://api.drupal.org/api/drupal/core%21modules%21node%21templates%21node-edit-form.html.twig/11.x

File: web/themes/custom/itkdev/itkdev_project_theme/templates/content-wrapper.html.twig [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      <h1 class="dark:text-white inline-block">
+        {% if view is defined %}
+          {{ view.title }}
+        {% elseif theme_hook_original is same as 'node' %}
+          {{ node.label }}
+        {% elseif theme_hook_original is same as 'user' %}
+          {{ content.field_name }}
+        {% elseif theme_hook_original is same as 'node_edit_form' %}
+          {{ form['#title'] }}
+        {% endif %}
```
[the remark this answers is not in the document]
Comment (martinyde) [reply web/themes/custom/itkdev/itkdev_project_theme/templates/content-wrapper.html.twig:13]: I committed the second proposal, but i might rewrite the whole thing at some point. I agree it seems a bit wobbly.

**Review by martinyde — APPROVED** [review]

**Pull request merged**